appveyor.yml

```diff
@@ -0,0 +1,25 @@
+# Should we cache local modules ?
+# cache:
+#  - node_modules
+
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+    - nodejs_version: "8"
```

Comment:
Do we actually need to test all node versions on Appveoyr as well ? I don't think it is worth it. AFAIK, Appveoyr is not as good as Travis at handling multiple jobs (especially regarding cache, even though you don't use it yet).